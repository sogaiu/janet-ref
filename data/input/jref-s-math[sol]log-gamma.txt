jref -s math/log-gamma
