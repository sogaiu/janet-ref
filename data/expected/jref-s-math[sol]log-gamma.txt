JANET_DEFINE_NAMED_MATHOP("log-gamma", lgamma, "Returns log-gamma(x).")

/*
   math/log-gamma
   +293 /home/user/src/janet/src/core/math.c
*/

